src/DistributedComputations/distributed_on_architecture.jl

```diff
@@ -2,6 +2,8 @@ using CUDA: CuArray
 using OffsetArrays
 import Oceananigans.Architectures: on_architecture
 
+import Oceananigans.Architectures: on_architecture
```

Comment:
```suggestion
import Oceananigans.Architectures: on_architecture
```

Reply:
what's the difference here in your suggestion @simone-silvestri -- I don't see any

Reply:
OK I figured it out! there was a duplicate import